src/frontend/next/src/pages/myfeeds.tsx

```diff
@@ -0,0 +1,216 @@
+import { useState, useEffect, useContext, FC, FormEvent } from 'react';
+import { useRouter } from 'next/router';
+import { Box, Card, Container, Grid, IconButton, Typography } from '@material-ui/core';
+import { AccountCircle, AddCircle, RssFeed } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+import { ValidatorForm, TextValidator } from 'react-material-ui-form-validator';
+import { isWebUri } from 'valid-url';
+
+import PageBase from './PageBase';
+import useSiteMetadata from '../hooks/use-site-metadata';
+import ExistingFeedList from '../components/MyFeedsPage/ExistingFeedList.jsx';
+import HelpPopoverButton from '../components/MyFeedsPage/HelpPopoverButton.jsx';
+import CustomizedSnackBar from '../components/SnackBar';
+import { UserStateContext } from '../contexts/User/UserContext';
+
+const useStyles = makeStyles((theme) => ({
+  margin: {
+    margin: theme.spacing(2),
+  },
+  button: {
+    padding: '3px 0 3px 0',
+  },
+}));
+
+// Type to determine what is a User
+interface User {
+  id: number;
+  name: string;
+}
+
+// Type to determine what is a FeedHash
+interface FeedHash {
+  [key: number]: {
+    author: string;
+    url: string;
+  };
+}
+
+// Type to determine what is a Feed
+interface Feed {
+  id: number;
+  author: string;
+  url: string;
+}
+
+interface MyFeedsProps {}
+
+const MyFeeds: FC<MyFeedsProps> = () => {
+  const router = useRouter();
+  const user: User = useContext(UserStateContext);
```

Comment:
This can get updated to use what I'm doing in #1521 once that lands:

```ts
import { useUser } from '../components/UserProvider';
...
const user = useUser();
```